### DISCUSSION TITLE
SSH using Putty

### DISCUSSION THREAD

**Discussion by Moekali84** [akash-network/community#370] [Q&A]
Hello,
 I have Created the deployment below and was successfully able to SSH into it and use it as intended for exactly three. times. now, eveytime i add my IP address to the Expose box and add an "ip name" my deployments never get any bids and it just keeps waiting. Can someone Please Help. (I do have a fake IP below Just Fyi)
 ---
version: "2.0"
services:
  web:
    image: opendronemap/nodeodm
    expose:
      - port: 3000
        as: 80
        accept:
          - 192.0.0.0
        to:
          - global: true
            ip: home
profiles:
  compute:
    web:
      resources:
        cpu:
          units: 128
        memory:
          size: 100GB
        storage:
          - size: 70Gi
  placement:
    dcloud:
      pricing:
        web:
          denom: uakt
          amount: 1000
deployment:
  web:
    dcloud:
      profile: web
      count: 1
endpoints:
  home:
    kind: ip

**Comment by piber-dev** [comment]
hello @Moekali84 

i believe the reason to why you are not receiving bids is that the providers that offers IP leases for the SDL i tried do not have enough resources for your SDL. [provider 1](https://deploy.cloudmos.io/providers/akash15ksejj7g4su7ljufsg0a8eglvkje94z8qsh68a) only has 105 available threads and [provider 2](https://deploy.cloudmos.io/providers/akash15tl6v6gd0nte0syyxnv57zmmspgju4c3xfmdhk) only 93 available threads. my SDL is the same as yours, only that i use 0.5 threads, 0.5 GB RAM, and 1 GB storage. 128 threads is not a requirement per the [installation docs of opendronemap](https://docs.opendronemap.org/installation/) so you should be fine to use fewer, where it looks like you will receive bids.

![image](https://github.com/akash-network/community/assets/138930882/0cdab3b6-5d7f-4a65-812d-6156025e5767)

**Reply by Moekali84** [reply]
i dont think thats true though. you can remove the IP request on the same requirement  and have tons of ppl provide bids. i was chatting with the Dev Team on discord for the past 4 days and they didnt seem to know about this issue. but they said it will be on their todo list in the coming weeks. which i really really hope they do fix it .
